src/components/HistoryTables/MsigProposals/Proposal.tsx

```diff
@@ -161,40 +168,30 @@ export default function ProposalDetail({
       <hr />
       <Lines>
         <Line label='Proposal ID'>{id}</Line>
-        <Line label='Proposer'>
-          {proposal?.approved[0] && (
-            <AddressLink
-              id={proposal.approved[0].id}
-              address={proposal.approved[0].robust}
-              hideCopyText={false}
-            />
-          )}
-        </Line>
-        <Line
-          label={`Approvers${
-            proposal?.approved ? ` (${proposal?.approved.length})` : ''
-          }`}
-        >
-          {proposal?.approved.map((approver: Address) => (
-            <AddressLink
-              key={approver.robust || approver.id}
-              id={approver.id}
-              address={approver.robust}
-            />
-          ))}
-        </Line>
+        <AddressLine label='Proposer' value={proposal.approved[0]} />
+        {proposal.approved.map((approver: Address, index) => (
+          <AddressLine
+            key={approver.robust || approver.id}
+            label={index === 0 ? `Approvers (${proposal.approved.length})` : ''}
+            value={approver}
+          />
+        ))}
         <Line label='Approvals until execution'>{approvalsUntilExecution}</Line>
-        <Parameters
-          params={{
-            params: {
-              to: proposal.to.robust,
-              value: proposal.value,
-              method: proposal.method,
-              params: proposal.params
-            }
-          }}
-          actorName='/multisig'
-          depth={0}
+        <hr />
+        <AddressLine label='To' value={proposal.to} />
+        <MethodLine
```

Comment:
I think we can just use a regular `Line` for this. The badges look kinda goofy i think